### PR TITLE
docs: add 3.26.10 changelog

### PR DESCRIPTION
**Pull request by chenshuai2144**
## 3.26.10

`2020-02-16`

- 🐞 Fix Input.Group inside `<Form layout="vertical" >` 1px bug. [#20685](https://github.com/ant-design/ant-design/pull/20685)
- 🐞 Fix Badge color not working when contains children. [#21333](https://github.com/ant-design/ant-design/pull/21333)
- 🐞 Fix Alert close button extra padding. [#21325](https://github.com/ant-design/ant-design/pull/21325)
- 🐞 Fix Affix throws `Cannot read property getBoundingClientRect in mobile device`. [#21350](https://github.com/ant-design/ant-design/pull/21350)

-----

## 3.26.10

`2020-02-16`

- 🐞 修复 `<Form layout="vertical" >` 内 Input.Group 偏上一像素的问题。[#20685](https://github.com/ant-design/ant-design/pull/20685)
- 🐞 修复 Badge 包裹模式下 `color` 属性失效的问题。[#21333](https://github.com/ant-design/ant-design/pull/21333)
- 🐞 修复 Alert 关闭按钮额外的 padding。[#21325](https://github.com/ant-design/ant-design/pull/21325)
- 🐞 修复 Affix 在移动设备下抛错 `Cannot read property getBoundingClientRect` 的问题。[#21350](https://github.com/ant-design/ant-design/pull/21350)

-----
[View rendered CHANGELOG.en-US.md](https://github.com/ant-design/ant-design/blob/3.26.10-changelog/CHANGELOG.en-US.md)
[View rendered CHANGELOG.zh-CN.md](https://github.com/ant-design/ant-design/blob/3.26.10-changelog/CHANGELOG.zh-CN.md)

### CODE REVIEW

**Comment by yoyo837** [comment]
#21391

**Comment by afc163** [comment]
https://github.com/ant-design/ant-design/pull/21306

**Comment by afc163** [comment]
#21310

**Comment by chenshuai2144** [comment]
@yoyo837 这个不是 Revert  了？

**Comment by afc163** [comment]
Revert 的 #21250 在 3.26.9 里发布了。

File: CHANGELOG.zh-CN.md [hunk 1]
```diff
@@ -34,6 +36,7 @@ timeline: true
 - 🐞 修复 Steps 在 `size="small"` 和 `labelPlacement="vertical"` 时图标没有对齐的问题。[#21258](https://github.com/ant-design/ant-design/pull/21258)
 - 🐞 修复 Typography 在可编辑状态时光标没有在输入框末位的问题。[#21268](https://github.com/ant-design/ant-design/pull/21268)
 - TypeScript
+
```
Comment (afc163) [inline CHANGELOG.zh-CN.md]: 别修改这个。

**Comment by yoyo837** [comment]
Revert 了 #21250，上一个版本发布的。

File: CHANGELOG.en-US.md [hunk 2]
```diff
@@ -15,6 +15,19 @@ timeline: true
...
+`2020-02-16`
+
+- 🐞 Fix Input.Group inside `<Form layout="vertical" >` 1px bug. [#20685](https://github.com/ant-design/ant-design/pull/20685)
+- 🐞 Fix Badge color not working when contains children. [#21333](https://github.com/ant-design/ant-design/pull/21333)
+- 🐞 Fix Alert close button extra padding. [#21325](https://github.com/ant-design/ant-design/pull/21325)
+- 🐞 Fix Affix throws `Cannot read property getBoundingClientRect in mobile device`. [#21350](https://github.com/ant-design/ant-design/pull/21350)
+- 💄 Tweak Steps 1px align issue. [#21306](https://github.com/ant-design/ant-design/pull/21306)
+- 💄 Fix Row component affect next element style issue. [#21310](https://github.com/ant-design/ant-design/pull/21310)
+- Typescript
+  - 🔷 Rollback [#21250](https://github.com/ant-design/ant-design/pull/21250) Type Definition Update.[#21356](https://github.com/ant-design/ant-design/pull/21356)
```
Comment (afc163) [inline CHANGELOG.en-US.md]: Revert

File: CHANGELOG.zh-CN.md [hunk 3]
```diff
@@ -15,6 +15,19 @@ timeline: true
 
 ---
 
+## 3.26.10
+
+`2020-02-16`
+
+- 🐞 修复 `<Form layout="vertical" >` 内 Input.Group 偏上一像素的问题。[#20685](https://github.com/ant-design/ant-design/pull/20685)
+- 🐞 修复 Badge 包裹模式下 `color` 属性失效的问题。[#21333](https://github.com/ant-design/ant-design/pull/21333)
+- 🐞 修复 Alert 关闭按钮额外的 padding。[#21325](https://github.com/ant-design/ant-design/pull/21325)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md]:
```suggestion
- 🐞 修复 Alert 关闭按钮额外的 `padding`。[#21325](https://github.com/ant-design/ant-design/pull/21325)
```

File: CHANGELOG.zh-CN.md [hunk 4]
```diff
@@ -15,6 +15,19 @@ timeline: true
...
+- 🐞 修复 Affix 在移动设备下抛错 `Cannot read property getBoundingClientRect` 的问题。[#21350](https://github.com/ant-design/ant-design/pull/21350)
+- 💄 微调 Steps 文本 1px 使其居中对齐。[#21306](https://github.com/ant-design/ant-design/pull/21306)
+- 💄 修复 Row 组件影响下一个元素样式问题。[#21310](https://github.com/ant-design/ant-design/pull/21310)
+- Typescript
+  - 🔷 回滚 [#21250](https://github.com/ant-design/ant-design/pull/21250)的类型定义更新。[#21356](https://github.com/ant-design/ant-design/pull/21356)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md]:
```suggestion
  - 🔷 回滚 [#21250](https://github.com/ant-design/ant-design/pull/21250) 的类型定义更新。[#21356](https://github.com/ant-design/ant-design/pull/21356)
```

**Pull request merged**